oursite/upoi/templates/upoi/index.html

```diff
@@ -347,4 +353,4 @@ <h1>Modo experto</h1>
     <noscript><p><img src="//elblogdehumitos.com/piwik/piwik.php?idsite=2" style="border:0;" alt="" /></p></noscript>
     <!-- End Piwik Code -->
   </body>
-</html>
+</html>
```

Comment:
Debes configurar tu editor para que siempre deje una línea extra al final del archivo.

Reply:
Esto me ha despertado un poco de curiosidad. 
Lo he cambiado en la configuración de mi VS Code, pero antes de hacerlo he consultado [esta discusión](https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file) al respecto. 
Aparte de lo de que ciertas herramientas, sobre todo más antiguas, podían dar problemas si no estaba esta línea, el argumento que más me ha convencido ha sido el de cómo actúa `diff` cuando se la encuentra o no. Ha sido interesante ver también el argumento expuesto por Mathias Bak con la "trailing comma" que me tenía algo perpleja. Prestaré más atención a ponerlas también de ahora en adelante. ¡Gracias!